File: BeeCrowd C++/Iniciante/1037.cpp
#include <iostream>
using namespace std;

int main(){
  float N;

  cin >> N;

  if (N >= 0 && N <= 25){
    cout << "Intervalo [0,25]" << endl;
  
  }else if (N > 25 && N <= 50){
    cout << "Intervalo (25,50]" << endl;
  
  }else if (N > 50 && N <= 75){
    cout << "Intervalo (50,75]" << endl;
  
  }else if (N > 75 && N <= 100){
    cout << "Intervalo (75,100]" << endl;
  
  }else{
    cout << "Fora de intervalo" << endl;
  }
  return 0;
}

File: BeeCrowd Python/Iniciante/1037.py
def main():
    num = float(input())

    if num < 0 or num > 100:
        print("Fora de intervalo")
    elif num >= 0 and num <= 25:
        print("Intervalo [0,25]")
    elif num <= 50:
        print("Intervalo (25,50]")
    elif num <= 75:
        print("Intervalo (50,75]")
    else:
        print("Intervalo (75,100]")
        
main()
